File: RQ2_Prompt_Elements/OpenAI_Data/HumanEvalPython_output/scenario2/id_102T2000Test.py
# id_102_Test.java
from scenario2.id_102 import choose_num

import pytest

class id_102_Test:
    """
    Test class of id_102.py.
    It contains ten unit test cases for the :func:`scenario2.id_102.choose_num`.
    """


    def test_choose_num_1(self):
        """
        Test case 1 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 1) == -1

    def test_choose_num_2(self):
        """
        Test case 2 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 2) == -1

    def test_choose_num_3(self):
        """
        Test case 3 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 3) == 2

    def test_choose_num_4(self):
        """
        Test case 4 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 4) == 4

    def test_choose_num_5(self):
        """
        Test case 5 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 5) == 4

    def test_choose_num_6(self):
        """
        Test case 6 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 6) == 6

    def test_choose_num_7(self):
        """
        Test case 7 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 7) == 6

    def test_choose_num_8(self):
        """
        Test case 8 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 8) == 8

    def test_choose_num_9(self):
        """
        Test case 9 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 9) == 8

    def test_choose_num_10(self):
        """
        Test case 10 for :func:`scenario2.id_102.choose_num`.
        """
        assert choose_num(1, 10) == 10

File: RQ2_Prompt_Elements/StarCoder_Data/HumanEvalJava_output/scenario2/id_102T2000Test.java
// ChooseNumTest.java
package scenario2;


import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

/**
* Test class of {@link ChooseNum}.
* It contains ten unit test cases for the {@link ChooseNum#chooseNum(int, int)} method.
*/
class ChooseNumTest {

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 10.
     */
    @Test
    void testChooseNum1() {
        assertEquals(10, ChooseNum.chooseNum(1, 10));
    }

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 11.
     */
    @Test
    void testChooseNum2() {
        assertEquals(-1, ChooseNum.chooseNum(1, 11));
    }

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 12.
     */
    @Test
    void testChooseNum3() {
        assertEquals(12, ChooseNum.chooseNum(1, 12));
    }

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 13.
     */
    @Test
    void testChooseNum4() {
        assertEquals(-1, ChooseNum.chooseNum(1, 13));
    }

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 14.
     */
    @Test
    void testChooseNum5() {
        assertEquals(14, ChooseNum.chooseNum(1, 14));
    }

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 15.
     */
    @Test
    void testChooseNum6() {
        assertEquals(-1, ChooseNum.chooseNum(1, 15));
    }

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 16.
     */
    @Test
    void testChooseNum7() {
        assertEquals(16, ChooseNum.chooseNum(1, 16));
    }

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 17.
     */
    @Test
    void testChooseNum8() {
        assertEquals(-1, ChooseNum.chooseNum(1, 17));
    }

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 18.
     */
    @Test
    void testChooseNum9() {
        assertEquals(18, ChooseNum.chooseNum(1, 18));
    }

    /**
     * Test case for {@link ChooseNum#chooseNum(int, int)}
     * when the first argument is 1 and the second argument is 19.
     */
    @Test
    void testChooseNum10() {
        assertEquals(-1, ChooseNum.chooseNum(1, 19));
    }
}
